scripts/pi-hole/js/utils.js

```diff
@@ -83,63 +83,51 @@ function padNumber(num) {
   return ("00" + num).substr(-2, 2);
 }
 
-var info = null;
+var showAlertBox = null;
 function showAlert(type, icon, title, message) {
-  var opts = {};
-  title = "&nbsp;<strong>" + title + "</strong><br>";
+  const options = {
```

Comment:
We use `const` here. I will details the *why* below in case anyone doesn't know (I will admit: I didn't know).

In JavaScript, `const` is used to declare a variable that cannot be reassigned. However, it does not make the variable immutable. This means that while you cannot reassign a `const` variable to a new value, you can still modify the properties of an object that is assigned to a `const` variable.

Here, the `options` and `settings` objects are declared using `const`. This means that you cannot reassign them to a new value. However, the properties of these objects can still be modified.

For example, the `options` and `settings` objects are modified based on the `type` parameter. The `icon` property of the `options` object is set to a default value if it is not provided, and the `delay` property of the `settings` object is multiplied by 2 if the `type` is "warning" or "error".

In summary, `const` in JavaScript does not make objects immutable, it only prevents the variable from being reassigned to a new value which seems a bit counter-intuitive given the name "const".